### ISSUE TITLE
CR remains in v0.1.2

### ISSUE THREAD

**Issue by iorate** [microsoft/license-checker-webpack-plugin#12]
In v0.0.9, CRLF (`'\r\n'`) in a license file is converted to LF (`'\n'`), but in v0.1.2, CRLF of a non-empty line is not.

For example, for `typescript@3.6.3`, which `LICENSE.txt` includes CRLF (Windows-style),

v0.0.9:
```
--------------------------------------------------------------------------------
typescript v3.6.3 - Microsoft Corp.
git+https://github.com/Microsoft/TypeScript.git
--------------------------------------------------------------------------------

Apache License

Version 2.0, January 2004
```

v0.1.2 (`^M` means CR):
```
--------------------------------------------------------------------------------
typescript v3.6.3 - Microsoft Corp.
git+https://github.com/Microsoft/TypeScript.git
--------------------------------------------------------------------------------

Apache License^M

Version 2.0, January 2004^M
```

This change seems to be related to update of `wrap-ansi`, used in `src/licenseUtils.js`.
`wrapAnsi('a\r\n\r\nb', 5)` returns `'a\n\nb'` in v4.0.0, but `'a\r\n\nb'` in v6.0.0.

I think this is a regression, but don't know whether it should be fixed by this package or by `wrap-ansi`.

**Comment by unindented** [comment]
I've opened an issue in the `wrap-ansi` project: https://github.com/chalk/wrap-ansi/issues/39

Let's see what they say.

**Comment by unindented** [comment]
Looks like they're going to fix this: https://github.com/chalk/wrap-ansi/pull/40

**Comment by unindented** [comment]
@iorate I've published `0.1.4` which solves this issue by upgrading `wrap-ansi` to `6.1.0`.

**Comment by iorate** [comment]
Thank you!
